crates/subspace-farmer/src/ws_rpc.rs

```diff
@@ -34,7 +34,7 @@ impl RpcClient for WsRpc {
             .await?)
     }
 
-    /// Get a block by number.
+    /// Get a block by number
```

Comment:
BTW, you don't need to copy comments on the impl, comments from trait will propagate here anyway.

Reply:
That's wonderful, fixing